### PR TITLE
#2973 Add U from UPu for InfiniteBus and Bus.

### PR DESCRIPTION
**Pull request by gautierbureau**
**Checklist before requesting a review**

*use `'[x]'` to check the checkboxes, or submit the PR and then click the checkboxes*

- [ ] unit tests and non-regression tests were added (new model, new feature and bug fix)
- [ ] main documentation was updated (update of input/output file, 3rd party, model, repository organization, solver)
- [ ] example documentations were updated (new example in examples folder)
- [ ] the corresponding milestone was added in the ticket and in this PR
- [ ] if this PR modifies the parameters or inputs/outputs of a model/solver: the corresponding xsl was added in util/xsl and platform DB were updated
- [ ] if this PR modifies a dictionary: the corresponding french dictionary was updated

### CODE REVIEW

**Review by rosiereflo — DISMISSED** [review]

File: dynawo/sources/Models/Modelica/Dynawo/Electrical/Buses/Bus.mo [hunk 1]
```diff
@@ -18,16 +18,20 @@ model Bus "Bus"
 
   extends AdditionalIcons.Bus;
 
+  parameter Types.VoltageModule UNom = 1.0 "Nominal voltage in kV";
+
   Connectors.ACPower terminal annotation(
     Placement(visible = true, transformation(origin = {-1.42109e-14, 98}, extent = {{-10, -10}, {10, 10}}, rotation = 0), iconTransformation(origin = {-1.42109e-14, 0}, extent = {{-10, -10}, {10, 10}}, rotation = 0)));
   Types.VoltageModulePu UPu "Voltage amplitude at terminal in pu (base UNom)";
+  Types.VoltageModulePu U "Voltage amplitude at terminal";
```
Comment (joyelfeghali) [inline dynawo/sources/Models/Modelica/Dynawo/Electrical/Buses/Bus.mo:26]: Types.VoltageModule U "Voltage amplitude at terminal in kV"

File: dynawo/sources/Models/Modelica/Dynawo/Electrical/Buses/InfiniteBus.mo [hunk 2]
```diff
@@ -27,15 +27,18 @@ model InfiniteBus "Infinite bus"
 
   parameter Types.PerUnit UPu "Infinite bus constant voltage module";
   parameter Types.Angle UPhase "Infinite bus constant voltage angle";
+  parameter Types.VoltageModule UNom = 1.0 "Nominal voltage in kV";
 
   Types.VoltageModulePu UPuVar "Voltage amplitude at terminal in pu (base UNom)";
   Types.Angle UPhaseVar "Voltage angle at terminal in rad";
+  Types.VoltageModulePu U "Voltage amplitude at terminal";
```
Comment (joyelfeghali) [inline dynawo/sources/Models/Modelica/Dynawo/Electrical/Buses/InfiniteBus.mo:34]: same

**Review by joyelfeghali — CHANGES_REQUESTED** [review]

File: dynawo/sources/Models/Modelica/Dynawo/Electrical/Buses/Bus.mo (hunk 1 above)
Comment (gautierbureau) [reply dynawo/sources/Models/Modelica/Dynawo/Electrical/Buses/Bus.mo]: done

File: dynawo/sources/Models/Modelica/Dynawo/Electrical/Buses/Bus.mo [hunk 3]
```diff
@@ -18,16 +18,20 @@ model Bus "Bus"
 
   extends AdditionalIcons.Bus;
 
+  parameter Types.VoltageModule UNom = 1.0 "Nominal voltage in kV";
+
   Connectors.ACPower terminal annotation(
     Placement(visible = true, transformation(origin = {-1.42109e-14, 98}, extent = {{-10, -10}, {10, 10}}, rotation = 0), iconTransformation(origin = {-1.42109e-14, 0}, extent = {{-10, -10}, {10, 10}}, rotation = 0)));
   Types.VoltageModulePu UPu "Voltage amplitude at terminal in pu (base UNom)";
+  Types.VoltageModule U "Voltage amplitude at terminal in kV"
```
Comment (joyelfeghali) [inline dynawo/sources/Models/Modelica/Dynawo/Electrical/Buses/Bus.mo:26]: semicolon missing here

File: dynawo/sources/Models/Modelica/Dynawo/Electrical/Buses/InfiniteBus.mo [hunk 4]
```diff
@@ -27,15 +27,18 @@ model InfiniteBus "Infinite bus"
 
   parameter Types.PerUnit UPu "Infinite bus constant voltage module";
   parameter Types.Angle UPhase "Infinite bus constant voltage angle";
+  parameter Types.VoltageModule UNom = 1.0 "Nominal voltage in kV";
 
   Types.VoltageModulePu UPuVar "Voltage amplitude at terminal in pu (base UNom)";
   Types.Angle UPhaseVar "Voltage angle at terminal in rad";
+  Types.VoltageModule U "Voltage amplitude at terminal in kV"
```
Comment (joyelfeghali) [inline dynawo/sources/Models/Modelica/Dynawo/Electrical/Buses/InfiniteBus.mo:34]: and here

**Review by joyelfeghali — CHANGES_REQUESTED** [review]

**Review by joyelfeghali — APPROVED** [review]

**Pull request merged**